managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -121,11 +121,15 @@ public UniverseController(YBClientService service) {
   }
 
   private boolean validateEncryption(ObjectNode formData) {
+    JsonNode clusters = formData.get("clusters");
+    if (clusters == null) {
+      return true;
+    }
     for (JsonNode cluster : formData.get("clusters")) {
       JsonNode nodeToNodeEncryption = cluster.get("userIntent").get("enableNodeToNodeEncrypt");
```

Comment:
This can still result in a NPE if a cluster does not have a userIntent field. This method should not assume the data is in any specific shape since we don't have upstream validation of the shape of the JSON request payload before this method is called.